src/Dynamo.All.2012.sln

```diff
@@ -107,11 +107,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DSOffice", "Libraries\DSOff
 		{133FC760-5699-46D9-BEA6-E816B5F01016} = {133FC760-5699-46D9-BEA6-E816B5F01016}
```

Comment:
2012 and 2013 solutions are not in sync anymore, either we delete 2012 sln if we don't want to support it further or maintain it. I think our CI still makes use of 2012 sln.